src/js/store/epics/reducer.ts

```diff
@@ -11,6 +11,7 @@ export interface Epic {
   old_slugs: string[];
   description: string;
   description_rendered: string;
+  task_count: bigint;
```

Comment:
I'm not sure it's worth using `bigint` here vs the usual `number`.

```suggestion
  task_count: number;
```